modules/ornl/frontier/rct/1.83.0.lua

```diff
@@ -0,0 +1,46 @@
+--[[
+
+    file rct module
+
+    Copyright 2024 RADICAL-Project
+
+]]--
+
+
+-- local vars --
+
+
+-- template variables ----------------------------------------------------------
+local SITE                = "ornl"
+local RESOURCE            = "frontier"
+
+local ROOT                = "/ccs/home/matitov/"
```

Comment:
let's use this path `/lustre/orion/world-shared/chm155/radical`

Reply:
Done.